### ISSUE TITLE
Adding Carthage support?

### ISSUE THREAD

**Issue by nickvelloff** [ide/WebASDKImageManager#1]
Hi. I can't currently use CocoaPods because there is an issue using the same dependency in multiple targets. This is required if you are using Share or Today extensions etc... 

I was wondering if there was any way this could be made Carthage compatible? The `SDWebImage` dependency is Carthage compatible already.

This would be a huge help as I'm have a really hard time caching images using `ASNetworkImageNode` and this looks like a great solution.
Thanks.

**Comment by ide** [comment]
Do you want to submit a PR? I haven't used Carthage so I'm not familiar with what it entails.

**Comment by nickvelloff** [comment]
Yes I will give it a shot. I have not created them yet either but I'm decently familiar with the requirements.
